docs/imageomics/treeoflife10m.md

```diff
@@ -1,9 +1,34 @@
-# Dataset Card for TreeOfLife-10M
+# How to Create TreeOfLife-10M
 
+**Note:** [TreeOfLife-10M](https://huggingface.co/datasets/imageomics/TreeOfLife-10M) has the EOL images, but not iNat21 or BIOSCAN-1M due to licensing restrictions. To reconstruct the _full_ dataset, please follow the steps outlined below in Reproduce TreeOfLife-10M.
```

Comment:
```suggestion
**Note:** 
- [TreeOfLife-10M](https://huggingface.co/datasets/imageomics/TreeOfLife-10M) has the EOL images, but not iNat21 or BIOSCAN-1M due to licensing restrictions. To reconstruct the _full_ dataset, please follow the steps outlined below in Reproduce TreeOfLife-10M.
- This reproduction process is designed to be run on an HPC system using Slurm.
```

Reply:
I think this would be a more logical division into bullet points:

> - [TreeOfLife-10M](https://huggingface.co/datasets/imageomics/TreeOfLife-10M) has the EOL images, but not iNat21 or BIOSCAN-1M due to licensing restrictions.
> - To reconstruct the _full_ dataset, please follow the steps outlined below in _Reproduce TreeOfLife-10M_. This reproduction process is designed to be run on an HPC system using Slurm.

(I also added emphasis to the section heading being referenced)